### ISSUE TITLE
Latest softhddevice (1:0.6.1rc1_63_g4fa4f66-6) kills vdr installation

### ISSUE THREAD

**Issue by connaisseur** [VDR4Arch/vdr4arch#137]
After latest "pacman -Suy" and a reboot "journalctl -lu vdr" says:

```
Mär 23 07:27:41 vdr1 vdr[525]: vdr: /usr/lib/vdr/plugins/libvdr-softhddevice.so.2.2.0: undefined symbol: avcodec_alloc_frame
Mär 23 07:27:41 vdr1 systemd[1]: vdr.service: Main process exited, code=killed, status=11/SEGV
Mär 23 07:27:41 vdr1 systemd[1]: Failed to start Video Disk Recorder.
```
Seems like latest patching / fiddling around ffmpeg finalizes the vdr for now!

**Comment by olebowle** [comment]
Same problem here with vdr-softhddevice-1:0.6.1rc1_63_g4fa4f66-**6**. However downgrading to vdr-softhddevice-1:0.6.1rc1_63_g4fa4f66-**5** works just fine. The debian ffmpeg2.9 patch doesn't seem to work properly.

**Comment by CReimer** [comment]
Fixed:  aeed27d
